assets/css/index.css

```diff
@@ -9,6 +9,30 @@ body {
   color: #333;
 }
 
+[data-theme="dark"] {
+  background-color: #131313;
```

Comment:
add a variable in `assets/css/variables.css` to value `#262626` and use it here for background color